<launch>

    <arg name="rviz" default="false" />
    <!-- <arg name="rviz" default="false" /> -->

    <node pkg="voxel_plus" type="lio_node" name="lio_node" output="screen">
        <rosparam command="load" file="$(find voxel_plus)/config/mapping.yaml" />
    </node>


    <group if="$(arg rviz)">
        <node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz" args="-d $(find voxel_plus)/rviz/default.rviz" />
    </group>
    
</launch>
